.github/pull_request_template.md

```diff
@@ -0,0 +1,72 @@
+# Description
+<!---  Write a brief description of what your code does and how it relates to the issue it is resolving or feature enhancement it is implementing. -->
+
+# Details
+#### URL to issue
+<!---
+  Delete this comment and include the URL of the issue the pull request is related to.
+  Refer to https://github.com/biomage-ltd/biomage-utils#stage on how to stage a staging environment.
```

Comment:
feels like this belongs to the `#### Link to staging deployment URL` section, not here under `#### URL to issue`

Reply:
moved